tests/sample/map_in_map_v2.c

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "bpf_helpers.h"
+
+#define INNER_MAP_ID 10
+
+SEC("maps")
+struct _ebpf_map_definition_in_file outer_map = {
+    .type = BPF_MAP_TYPE_ARRAY_OF_MAPS,
+    .key_size = sizeof(uint32_t),
+    .value_size = sizeof(uint32_t),
+    .max_entries = 1,
+    // inner_id refers to the id of the inner map in the same ELF object.
+    .inner_id = INNER_MAP_ID}; // (uint32_t)&inner_map
+
+SEC("maps")
+struct _ebpf_map_definition_in_file inner_map = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(uint32_t),
+    .value_size = sizeof(uint32_t),
+    .max_entries = 1,
+    .id = INNER_MAP_ID};
+
+SEC("xdp_prog") int lookup(struct xdp_md* ctx)
+{
+    uint32_t outer_key = 0;
+    void* nolocal_lru_map = bpf_map_lookup_elem(&outer_map, &outer_key);
```

Comment:
why use lru here ? Neither of the map types are LRU. Or is lru used in some other context?

Reply:
This is just wrong naming of variables. Will fix it.

Reply:
Fixed.